cpp/CMakeLists.txt

```diff
@@ -322,7 +322,7 @@ endif(PER_THREAD_DEFAULT_STREAM)
 
 if(BUILD_TESTS)
     include(CTest)
-    include(ConfigureGoogleTest)
+    find_package(GTest REQUIRED)
```

Comment:
So what happens if I'm not in a conda env that has gtest. We no longer include ConfigureGoogleTest and so it fails? 

We need to fall back to fetching and building gtest (as before) when it is not found.

Reply:
OK. I'll check and if fail use the existing configuration.

Reply:
You can follow the `Arrow` section above where we also have this pattern, and I'd be interested in handling `GTest` similarly in its own section

Reply:
We should also similarly handle google benchmark in the same way, there's conda packages here: https://anaconda.org/conda-forge/benchmark